# Team Report - COVID19 Dataset
The COVID-19 dataset is provided by the Cases, Deaths, and the County population.  
In this notebook, we will analyze the COVID-19 Dataset to fully understand it.

In [12]:
import pandas as pd

Let's import the Cases, Deaths, and County population data using pandas

In [13]:
confirmed = pd.read_csv("covid_confirmed_usafacts.csv")
deaths = pd.read_csv("covid_deaths_usafacts.csv")
county_population = pd.read_csv("covid_county_population_usafacts.csv")

## Cases Data

In [14]:
confirmed.head()

countyFIPS            County Name State  StateFIPS  2020-01-22  2020-01-23  \
0           0  Statewide Unallocated    AL          1           0           0   
1        1001        Autauga County     AL          1           0           0   
2        1003        Baldwin County     AL          1           0           0   
3        1005        Barbour County     AL          1           0           0   
4        1007           Bibb County     AL          1           0           0   

   2020-01-24  2020-01-25  2020-01-26  2020-01-27  ...  2023-07-14  \
0           0           0           0           0  ...           0   
1           0           0           0           0  ...       19913   
2           0           0           0           0  ...       70521   
3           0           0           0           0  ...        7582   
4           0           0           0           0  ...        8149   

   2023-07-15  2023-07-16  2023-07-17  2023-07-18  2023-07-19  2023-07-20  \
0           0           0           0           0           0           0   
1       19913       19913       19913       19913       19913       19913   
2       70521       70521       70521       70521       70521       70521   
3        7582        7582        7582        7582        7582        7582   
4        8149        8149        8149        8149        8149        8149   

   2023-07-21  2023-07-22  2023-07-23  
0           0           0           0  
1       19913       19913       19913  
2       70521       70521       70521  
3        7582        7582        7582  
4        8149        8149        8149  

[5 rows x 1269 columns]

We can see 4 columns.  
Additionally, there is a column for every day starting from the 01/22/2020 to 07/23/2023 with the number of confirmed covid cases.  

Let's have a closer look at each Column...

### Variable dictionary for the Cases data

|    Name        | Definition                     | Data type | Possible values                           |Required?|
|:--------------:|:------------------------------:|:---------:|:-----------------------------------------:|:-------:|
| County FIPS    | Unique identifier for a County | Integer   | 0, 1001, 20075                            | Yes     |
| County Name    | Name of the County             | String    | "Barbour County", "Statewide Unallocated" | Yes     |
| State          | Name of the State              | String    | "AL, "KS", "WI"                           | Yes     |
| State FIPS     | Unique identifier for a State  | Integer   | 1, 5, 20                                  | Yes     |

## Deaths Data

In [15]:
deaths.head()

countyFIPS            County Name State  StateFIPS  2020-01-22  2020-01-23  \
0           0  Statewide Unallocated    AL          1           0           0   
1        1001        Autauga County     AL          1           0           0   
2        1003        Baldwin County     AL          1           0           0   
3        1005        Barbour County     AL          1           0           0   
4        1007           Bibb County     AL          1           0           0   

   2020-01-24  2020-01-25  2020-01-26  2020-01-27  ...  2023-07-14  \
0           0           0           0           0  ...           0   
1           0           0           0           0  ...         235   
2           0           0           0           0  ...         731   
3           0           0           0           0  ...         104   
4           0           0           0           0  ...         111   

   2023-07-15  2023-07-16  2023-07-17  2023-07-18  2023-07-19  2023-07-20  \
0           0           0           0           0           0           0   
1         235         235         235         235         235         235   
2         731         731         731         731         731         731   
3         104         104         104         104         104         104   
4         111         111         111         111         111         111   

   2023-07-21  2023-07-22  2023-07-23  
0           0           0           0  
1         235         235         235  
2         731         731         731  
3         104         104         104  
4         111         111         111  

[5 rows x 1269 columns]

Similarly to Cases, we can see 4 columns.  
Additionally, there is a column for every day starting from the 01/22/2020 to 07/23/2023 with the number of confirmed covid deaths. 

Let's have a closer look at each Column...

### Variable dictionary for the Deaths data

|    Name        | Definition                     | Data type | Possible values                           |Required?|
|:--------------:|:------------------------------:|:---------:|:-----------------------------------------:|:-------:|
| County FIPS    | Unique identifier for a County | Integer   | 0, 1001, 20075                            | Yes     |
| County Name    | Name of the County             | String    | "Barbour County", "Statewide Unallocated" | Yes     |
| State          | Name of the State              | String    | "AL, "KS", "WI"                           | Yes     |
| State FIPS     | Unique identifier for a State  | Integer   | 1, 5, 20                                  | Yes     |

## County Population Data

In [16]:
county_population.head()

countyFIPS            County Name State  population
0           0  Statewide Unallocated    AL           0
1        1001         Autauga County    AL       55869
2        1003         Baldwin County    AL      223234
3        1005         Barbour County    AL       24686
4        1007            Bibb County    AL       22394

Similarly to confirmed cases, we can see 4 columns.  
Intersetingly, the column StateFIPS does not exists in this dataset.  
Additionally, there is a column  with the population of the county.  

Let's have a closer look at each Column...

### Variable dictionary for the County Population data

|    Name        | Definition                     | Data type | Possible values                           |Required?|
|:--------------:|:------------------------------:|:---------:|:-----------------------------------------:|:-------:|
| County FIPS    | Unique identifier for a County | Integer   | 0, 1001, 20075                            | Yes     |
| County Name    | Name of the County             | String    | "Barbour County", "Statewide Unallocated" | Yes     |
| State          | Name of the State              | String    | "AL, "KS", "WI"                           | Yes     |
| Population     | The population of a County     | Integer   | 0, 8111, 164542                           | Yes     |